docs-translations/ko-KR/api/session.md

```diff
@@ -77,7 +77,7 @@ session.defaultSession.on('will-download', (event, item, webContents) => {
 
 `Session` 객체는 다음과 같은 메서드와 속성을 가지고 있습니다:
 
-#### `ses.cookies`
+### `ses.cookies`
```

Comment:
I just realized these are changes in doc-translations, @zeke I think you forgot to revert the changes there.